csrankings.csv

```diff
@@ -21493,4 +21493,4 @@ m. c. schraefel,University of Southampton,https://www.ecs.soton.ac.uk/people/sig
 Öznur Tastan,Sabancı University,http://people.sabanciuniv.edu/otastan,8ljjXmEAAAAJ
 Öznur Özkasap,Koç University,http://home.ku.edu.tr/~oozkasap,v2WiJeIAAAAJ
 Ümit V. Çatalyürek,Georgia Institute of Technology,https://www.cc.gatech.edu/~umit,OLDMURQAAAAJ
-Xinyu Wang,University of Michigan,https://web.eecs.umich.edu/~xwangsd/,mvasb5AAAAAJ
+Xinyu Wang 006,University of Michigan,https://web.eecs.umich.edu/~xwangsd/,mvasb5AAAAAJ

```

Comment:
The correct suffix is 0006 (https://dblp.org/pid/68/1277-6.html). Also, please place in alphabetical order.